ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Sandboxed/BraveTranslateScriptHandler.swift

```diff
@@ -14,12 +14,12 @@ protocol BraveTranslateScriptHandlerDelegate: NSObject {
   func updateTranslateURLBar(tab: Tab?, state: TranslateURLBarButton.TranslateState)
   func canShowTranslateOnboarding(tab: Tab?) -> Bool
   func showTranslateOnboarding(tab: Tab?, completion: @escaping (_ translateEnabled: Bool?) -> Void)
-  func presentToast(tab: Tab?, languageInfo: BraveTranslateLanguageInfo)
+  func presentTranslateToast(tab: Tab?, languageInfo: BraveTranslateLanguageInfo)
+  func presentTranslateError(tab: Tab?)
```

Comment:
All of these don't make sense taking optional Tab's, should ensure a tab exists before calling the the delegate methods